setup.py

```diff
@@ -58,7 +58,6 @@ def get_all_extras() -> Dict:
     "valory-docker-compose==1.29.3",
     "werkzeug==2.0.3",
     "docker==6.1.2",
-    "hexbytes",
```

Comment:
Temporary change to test the safeguard, It works https://github.com/valory-xyz/open-autonomy/actions/runs/6182364864/job/16781954535?pr=2055